### PR TITLE
Added Request and Response headers

### PR DESCRIPTION
**Pull request by DivyaDosibhatla**
Added Request and Response headers

### CODE REVIEW

File: java/src/main/java/com/cybersource/ws/client/Connection.java [hunk 1]
```diff
@@ -126,6 +126,8 @@ private void checkForFault()
             logger.log(Logger.LT_INFO, "Reading response...");
 
             int responseCode = getHttpResponseCode();
+            
+            logResponseHeaders();
```
Comment (mahendya1002) [inline java/src/main/java/com/cybersource/ws/client/Connection.java]: remove this line. why this is require here, you have already kept those two methods below

File: java/src/main/java/com/cybersource/ws/client/HttpClientConnection.java [hunk 2]
```diff
@@ -231,6 +237,28 @@ public boolean retryMethod(
             return false;
         }
     }
+    
+	@Override
+	void logRequestHeaders() {
+		Header[] headers=postMethod.getRequestHeaders();
```
Comment (mahendya1002) [inline java/src/main/java/com/cybersource/ws/client/HttpClientConnection.java]: there is no need to convert to map. can we directly print headers ?

File: java/src/main/java/com/cybersource/ws/client/HttpClientConnection.java [hunk 3]
```diff
@@ -231,6 +237,28 @@ public boolean retryMethod(
...
+			headerMap.put(header.getName(), header.getValue());
+		 }
+        	logger.log(Logger.LT_INFO, "Request Headers: " + headerMap);
+	    }
+	
+	@Override
+	public void logResponseHeaders() {
+		Header[] headers = postMethod.getResponseHeaders();
+		 Map<String, String> headerMap = new HashMap<String, String>();
+		 for(Header header: headers){
```
Comment (mahendya1002) [inline java/src/main/java/com/cybersource/ws/client/HttpClientConnection.java]: same as prev comment.

Comment (DivyaDosibhatla) [reply java/src/main/java/com/cybersource/ws/client/HttpClientConnection.java]: headers array has key and value pairs.If im printing just headers, then the object is getting printed. Removed the map and printed as ArrayList.

File: java/src/main/java/com/cybersource/ws/client/HttpClientConnection.java (hunk 2 above)
Comment (DivyaDosibhatla) [reply java/src/main/java/com/cybersource/ws/client/HttpClientConnection.java]: headers array has key and value pairs.If im printing just headers, then the object is getting printed. Removed the map and printed as ArrayList.

File: java/src/main/java/com/cybersource/ws/client/Connection.java (hunk 1 above)
Comment (DivyaDosibhatla) [reply java/src/main/java/com/cybersource/ws/client/Connection.java]: removed extra lines

File: java/pom.xml [hunk 4]
```diff
@@ -9,7 +9,7 @@
     </parent>
     <groupId>com.cybersource</groupId>
     <artifactId>cybersource-sdk-java</artifactId>
-    <version>6.2.8-SNAPSHOT</version>
+    <version>6.2.9-SNAPSHOT</version>
```
Comment (mahendya1002) [inline java/pom.xml]: this will be 6.2.8 only

File: pom.xml [hunk 5]
```diff
@@ -12,7 +12,7 @@
 
     <groupId>com.cybersource</groupId>
     <artifactId>cybersource-sdk-master</artifactId>
-    <version>6.2.8-SNAPSHOT</version>
+    <version>6.2.9-SNAPSHOT</version>
```
Comment (mahendya1002) [inline pom.xml]: this will be 6.2.8-SNAPSHOT only

File: zip/pom.xml [hunk 6]
```diff
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>cybersource-sdk-master</artifactId>
         <groupId>com.cybersource</groupId>
-        <version>6.2.8-SNAPSHOT</version>
+        <version>6.2.9-SNAPSHOT</version>
```
Comment (mahendya1002) [inline zip/pom.xml]: keep 6.2.8-SNAPSHOT only

File: zip/pom.xml [hunk 7]
```diff
@@ -42,7 +42,7 @@
         <dependency>
             <groupId>com.cybersource</groupId>
             <artifactId>cybersource-sdk-java</artifactId>
-            <version>6.2.8-SNAPSHOT</version>
+            <version>6.2.9-SNAPSHOT</version>
```
Comment (mahendya1002) [inline zip/pom.xml]: keep 6.2.8-SNAPSHOT only

File: java/pom.xml (hunk 4 above)
Comment (DivyaDosibhatla) [reply java/pom.xml]: made it to 6.2.8-SNAPSHOT

File: pom.xml (hunk 5 above)
Comment (DivyaDosibhatla) [reply pom.xml]: made it to 6.2.8-SNAPSHOT

File: zip/pom.xml (hunk 6 above)
Comment (DivyaDosibhatla) [reply zip/pom.xml]: made it to 6.2.8-SNAPSHOT

**Pull request merged**